python/pyspark/ml/clustering.py

```diff
@@ -99,23 +99,13 @@ def numIter(self):
 @inherit_doc
 class _GaussianMixtureParams(HasMaxIter, HasFeaturesCol, HasSeed, HasPredictionCol,
                              HasProbabilityCol, HasTol, HasAggregationDepth, HasWeightCol,
-                             HasBlockSize):
+                             HasBlockSize, HasK):
```

Comment:
sounds like more meaningful name instead of short name: HasK. say, HasNumOfClusters?

Reply:
Thanks for your comment.  Unfortunately we can't change to ```HasNumOfClusters``` because  ```k``` /```getK```/```setK``` were used in the previous releases.